infinite_games/base/validator.py

```diff
@@ -496,7 +496,7 @@ def send_event_scores(self, miner_scores=None):
         else:
             bt.logging.debug('*** Grafana logs sent')
 
-    @backoff.on_exception(backoff.expo, Exception, max_tries=6)
+    # @backoff.on_exception(backoff.expo, Exception, max_tries=6)
```

Comment:
was this removed on purpopse? I think exponential backoff could be good here

Reply:
> it was creating a lot of problems by hiding logs recently

That sounds strange, do you have some example? If you log in the except branch before reraising the error, do you miss the logs? I see in the documentation some instructions for logging
https://pypi.org/project/backoff/
Also has some arg for additional logic on backoff `on_backoff=backoff_hdlr`

> because we will retry same intervals from logic anyway
if that retry logic was not unit tested, I would very much prefer using stuff from the standard or popular libraries